### ISSUE TITLE
VO+Safari "button" HTML summary of issues

### ISSUE THREAD

**Issue by shelbyelavsky** [fizzstudio/svg-aria-test#68]
This is an issue in reference to #23 which is related to the **"primitives and nested role=button"** branch.

<img width="753" alt="Screenshot of VoiceOver in test environment. A blue circle is highlighting Text Viewer text that reads 'toggle button, new line'." src="https://user-images.githubusercontent.com/98043076/161115492-89c780af-d2f6-470d-8bfd-6447acdf3ce9.png">

### Issue Summary
Issue 1: VoiceOver reads “Path 1” as a text element instead of what should be labels of the child element. (See another example in #66 issue.)

Issue 2: VoiceOver does not recognize the toggleable element as expected. It should be connected to the “path” element, but reads as ‘toggle button new line.’

Issue 3: VoiceOver cannot navigate to the remaining elements (“3 Text”), the next heading, or the text after “Path 1” toggle button. It navigates between “Path 1” or “toggle button new line” endlessly.

### Video of issue 
Issue 1 Example starts at 00:01:38
Issue 2 Example starts at 00:01:40
Issue 3 Example starts at 00:01:51
https://user-images.githubusercontent.com/98043076/161101206-0c294fa7-fa83-40ab-bcc3-42e1db6c43d2.mp4

### Steps to reproduce
Starting at the "html examples with toggleable state" heading: Use VoiceOver down arrow key to navigate into the HTML space. Continue navigating through and interact with the elements. 

### Expected behavior (Pass/Fail)
We aren't actually testing HTML in this project, but if we are comparing the SVG to HTML elements in how they perform we need to note the issues found.

### Technical Details
VoiceOver (Default)
Safari 15.1 (17612.2.9.1.20)
MacBook Pro/ macOS Monterey 12.0.1
